packages/frontend/.env

```diff
@@ -0,0 +1,2 @@
+VUE_APP_SERVER_URI=http://localhost:3000
+NODEJS_SERVER_PORT=3000
```

Comment:
probably not needed in frontend anymore, right?
backend should use `NODE_JS_SERVER_PORT`, frontend should use `VUE_APP_SERVER_URI`